SKYKitChat/Classes/SKYChatExtension.m

```diff
@@ -591,7 +591,11 @@ - (void)markLastReadMessage:(SKYMessage *)message
          inUserConversation:(SKYUserConversation *)userConversation
                  completion:(SKYChatUserConversationCompletion)completion
 {
-    userConversation.lastReadMessageID = [SKYReference referenceWithRecord:message];
+    if (message) {
```

Comment:
Thanks.

If `message` is nil, `message.recordID.recordName` is evaluated to nil, you might as well simply say:

```
userConversation.lastReadMessageID = message.recordID.recordName;
```